tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1898,6 +1897,21 @@ public function sendShareSpaceRequest(
 		);
 	}
 
+	/**
+	 * @When /^user "([^"]*)" changes the role of user "([^"]*)" to "([^"]*)" for space "([^"]*)"$/
```

Comment:
there is already a step for the func. `sendShareSpaceRequest` imo, u can reuse that.